Yoti.Auth.Sandbox/DocScan/Request/SandboxMedia.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Net.Http.Headers;
+
+namespace Yoti.Auth.Sandbox.DocScan.Request.Resource
+{
+    public class SandboxMedia
```

Comment:
This should live with the integration tests (web client)

Reply:
Moved in ddbb848